src/app/pcpolicy/pcpolicy.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="header">
+	<app-header (togglenavbar)="nav.toggle()"> </app-header>
+</div>
+<div class="container-fluid">
+	<app-navbar #nav (toggleposition)= " toggle() "> </app-navbar>
+	<div class="row">
+		<div [className]="position + ' pc_policy_body text-center'">
```

Comment:
No that doesn't work, it just takes one class variable. I initially tried doing it and it ignored the class = ""